src/translate_localization_files.py

```
@@ -1099,25 +1110,23 @@ def validate_paths(input_folder: str, translation_queue: str, translated_queue:
     logging.info("All critical paths are valid and accessible.")
 
 
-def get_changed_translation_files(input_folder_path: str, repo_root: str) -> List[str]:
+def get_changed_translation_files(target_repo_path: str) -> list[str]:
```

Ruff (0.12.2)

Unused function argument: `target_repo_path`

(ARG001)